worker/lease/doc.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package lease is a worker that manages leases, which are used by individual
+// juju workers.
+// The manager works by using these operations:
+//   - claim
+//   - revoke
+//   - pin
+//   - query
+//   - block
```

Comment:
Just a thought on structure - would it make more sense to have the descriptions inside the list, rather than listing them all and then describing them in a follow up paragraph? For example
```
// The manager works by using these operations:
//   - claim: Workers will claim a lease, and they are either attributed or blocked. The
// lease manager will keep track of all the blocked claims. ...
//   - revoke: ...
//   - ...
```